### PR TITLE
Update CirrusMDSDK to 5.X.X

### PR DESCRIPTION
**Pull request by taylorjcase**
## Description
Updated CirrusMDSDK to 5.X.X. Available as 5.0.0, 5.1.0, 5.20, 5.30, and 5.40. All three releases are the same code just compiled with three different versions of Xcode to help with compatibility in parent applications.

## Motivation and Context
Version 5.X.X of the CirrusMDSDK contains new features, UX/UI improvements, and bug fixes. A full list of the changes can be viewed in the CHANGELOG.

## Types of changes

- [x] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:

- [x] I have added tests to cover my changes.
- [ ] No tests are required for this change.
- [x] My change requires a documentation change.
- [x] My change requires a CHANGELOG update.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -533,6 +430,135 @@ CirrusMDSDK.singleton.onRemoteNotificationSelected(streamId: String, eventType:
...
+##### Obective-C
+
+```obj-c
+CirrusMDSDKConfig* config = [[CirrusMDSDKConfig alloc] init];
+config.enableDependents = YES;
+
+[CirrusMDSDK.singleton setConfig:config];
+```
+
+### Enable User Log Out
```
Comment (taylorjcase) [inline README.md:538]: Moved some documentation around so the README diff isn't great. This is documentation for the new enableUserLogOut config option.

File: README.md [hunk 2]
```diff
@@ -563,6 +589,28 @@ CirrusMDChannel* channel = [CirrusMDSDK.singleton channels][index];
 [CirrusMDSDK.singleton navigateToChannelWithId: channel.id];
 ```
 
+### User Agent Prefix
```
Comment (taylorjcase) [inline README.md:592]: This is also net new documentation for the new userAgentPrefix config option.

File: README.md [hunk 3]
```diff
@@ -571,28 +619,32 @@ The CirrusMDSDK can you provide you with a view controller that displays various
 
 THIS VIEW CONTROLLER SHOULD NEVER BE PRESENTED TO AN END USER. It is for debugging purposes only.
 
+The debug view controller can be accessed and presented manually or it can be accessed in Settings if `showDebugInSettings` is set to true on your `CirrusMDSDKConfig`
```
Comment (taylorjcase) [inline README.md:622]: Also net new documentation here around the new showDebugInSettings config option.

**Review by DanielGalarza — APPROVED** [review]
👍

**Pull request merged**